src/main/java/duke/handler/FileHandler.java

```diff
@@ -0,0 +1,130 @@
+package duke.handler;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Tasklist;
+import duke.task.Todo;
```

Comment:
——— For PR review ———
Nice and clear use of imports, even when import `duke.task.*` is tempting to write when you are clearly using all every component inside your `task` package.
Individual imports helps readability.